packages/lesswrong/components/votes/VoteAgreementIcon.tsx

```diff
@@ -159,60 +101,78 @@ const VoteAgreementIcon = ({
   bigVoteCompleted,
   alwaysColored,
   classes,
-}: VoteArrowIconProps & {
-  classes: ClassesType
+}: BaseVoteArrowIconProps & {
+  classes: ClassesType<typeof styles>
 }) => {
-  const theme = useTheme();
   const upOrDown = orientation === "left" ? "Downvote" : "Upvote"
   
-  const PrimaryIcon =  (upOrDown === "Downvote") ? ClearIcon : CheckIcon
-  const primaryIconStyling = (upOrDown === "Downvote") ? classes.clear : classes.check
-  
-  const BigVoteAccentIcon = (upOrDown === "Downvote") ? UpArrowIcon: CheckIcon
-  const bigVoteAccentStyling = (upOrDown === "Downvote") ? classes.smallArrowBigVoted : classes.smallCheckBigVoted
-  const bigVoteCompletedStyling = (upOrDown === "Downvote") ? classes.bigClearCompleted : classes.bigCheckCompleted
-  const bigVoteStyling = (upOrDown === "Downvote") ? classes.bigClear : classes.bigCheck
+  const primaryIcon =  (upOrDown === "Downvote") ? "CrossReaction" : "TickReaction"
+  const bigVoteAccentIcon = (upOrDown === "Downvote") ? "CrossReactionCap" : "TickReaction"
 
-  if (!enabled) {
-    eventHandlers = {};
-  }
+  const handlers = enabled ? eventHandlers : {};
 
   const {mainColor, lightColor} = useVoteColors(color);
 
+  const { ForumIcon } = Components;
+
+  const strongVoteLargeIconClasses = (upOrDown === "Downvote")
+    ? classNames(
+      bigVotingTransition && classes.entering,
+      classes.bigClear,
+      (bigVotingTransition || bigVoteCompleted || bigVoted) && classes.bigClearCompleted,
```

Comment:
`bigVotingTransition || bigVoteCompleted || bigVoted` seems like it can be extracted into a variable (given its frequent reuse).